src/main/java/com/amazon/kinesis/streaming/agent/config/AgentConfiguration.java

```diff
@@ -31,11 +31,12 @@
     static final boolean DEFAULT_CW_EMIT_METRICS = true;
     static final boolean DEFAULT_CW_TAG_INSTANCE = false;
     static final long DEFAULT_CW_BUFFER_TIME_MILLIS = 30_000L;
+    static final int DEFAULT_CW_METRIC_GRANULARITY_SECONDS = 60;
     static final boolean DEFAULT_LOG_EMIT_METRICS = false;
     static final boolean DEFAULT_LOG_EMIT_INTERNAL_METRICS = false;
     static final int DEFAULT_LOG_STATUS_REPORTING_PERIOD_SECONDS = 30;
     static final int DEFAULT_CHECKPOINT_TTL_DAYS = 7;
-    protected static final int DEFAULT_ASSUME_ROLE_DURATION_SECONDS = 
+    protected static final int DEFAULT_ASSUME_ROLE_DURATION_SECONDS =
```

Comment:
@chaochenq, Ideally there would be a document somewhere in the codebase where we can centralize all these settings and what they are used for. Do you want me to start that in the `README.md`?